services/kv_store.go

```diff
@@ -112,7 +111,20 @@ func BatchSet(kvs *KVPairs) error {
 	return err
 }
 
-/*GenKvStoreKey returns the key for inserting to KV-Store for data_maps.*/
-func GenKvStoreKey(gensisHash string, chunkIdx int) string {
-	return fmt.Sprintf("%s_%d", gensisHash, chunkIdx)
+func BatchDelete(ks *KVKeys) (err error) {
```

Comment:
can you revert this. Since I will have mine ready

Reply:
yeah, didn't notice you already did this

Reply:
Also this will break if too many pending transaction